RockWeb/Blocks/Event/RegistrationInstanceDetail.ascx

```diff
@@ -89,11 +89,13 @@
                                 <Rock:RockLiteral ID="lName" runat="server" Label="Name" />
                                 <Rock:RockLiteral ID="lMaxAttendees" runat="server" Label="Maximum Attendees" />
                                 <Rock:RockLiteral ID="lWorkflowType" runat="server" Label="Registration Workflow" />
+                                <Rock:RockLiteral ID="lStartDate" runat="server" Label="Registration Starts" />
                             </div>
                             <div class="col-md-6">
                                 <Rock:RockLiteral ID="lCost" runat="server" Label="Cost" />
                                 <Rock:RockLiteral ID="lMinimumInitialPayment" runat="server" Label="Minimum Initial Payment" />
                                 <Rock:RockLiteral ID="lAccount" runat="server" Label="Account" />
+                                <Rock:RockLiteral ID="lEndDate" runat="server" Label="Registration Ends" />
```

Comment:
@shivambareria , Instead of adding the lStartDate and lEndDate controls to the bottom of each column in existing row, can you add a new row (with two col-md-6 columns) just for these controls so that when browser is resized they do not appear so far away from each other:
![registration instance rock rms](https://user-images.githubusercontent.com/548180/28188762-2481a180-67d8-11e7-8c3d-1b0fbf9d64aa.png)